router/router.go

```diff
@@ -228,12 +230,10 @@ func New(cfg *config.Configuration, rateConvertor *currencies.RateConverter) (r
 	}
 
 	p, _ := filepath.Abs(infoDirectory)
-	bidderInfos := adapters.ParseBidderInfos(cfg.Adapters, p, openrtb_ext.BidderList())
+	bidderInfos := adapters.ParseBidderInfos(cfg.Adapters, p, openrtb_ext.BidderNames())
 
-	disabledBidders := map[string]string{
-		"indexExchange": "Bidder \"indexExchange\" has been deprecated and is no longer available. Please use bidder \"ix\" and note that the bidder params have changed.",
```

Comment:
Moved to `GetDisabledBiddersErrorMessages`. Do we really still need this though? How long ago was the rename?

Reply:
I think I am okay with dropping this, it has been at least a year or two now.